synapseclient/client.py

```diff
@@ -3870,22 +4235,28 @@ def _get_contributors(self, evaluation_id, team):
     @tracer.start_as_current_span("Synapse::_allowParticipation")
     def _allowParticipation(
         self,
-        evaluation,
-        user,
-        rights=["READ", "PARTICIPATE", "SUBMIT", "UPDATE_SUBMISSION"],
-    ):
+        evaluation: Union[Evaluation, str],
+        user: str,
+        rights: list = ["READ", "PARTICIPATE", "SUBMIT", "UPDATE_SUBMISSION"],
+    ) -> Dict[str, Union[str, list]]:
         """
         Grants the given user the minimal access rights to join and submit to an Evaluation.
         Note: The specification of this method has not been decided yet, so the method is likely to change in future.
 
-        :param evaluation: An Evaluation object or Evaluation ID
-        :param user:       Either a user group or the principal ID of a user to grant rights to.
-                           To allow all users, use "PUBLIC".
-                           To allow authenticated users, use "AUTHENTICATED_USERS".
-        :param rights:     The access rights to give to the users.
-                           Defaults to "READ", "PARTICIPATE", "SUBMIT", and "UPDATE_SUBMISSION".
-        """
+        Arguments:
+            evaluation: An Evaluation object or Evaluation ID
+            user:       Either a user group or the principal ID of a user to grant rights to.
+                        To allow all users, use "PUBLIC".
```

Comment:
These might look better in an unordered list (Same for `rights`)

Reply:
Done